typing_extensions/src_py3/test_typing_extensions.py

```diff
@@ -1188,7 +1188,7 @@ class E:
             self.assertIsInstance(E(), D)
 
 
-if NO_PROTOCOL:
+if not HAVE_PROTOCOLS:
```

Comment:
Arguably the entire `class ProtocolTests` should be inside `if HAVE_PROTOCOLS:` instead of this little block.